gcp-go-gke/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi-gcp/sdk/v2/go/gcp/container"
+	appsv1 "github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes/apps/v1"
+	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes/core/v1"
+	metav1 "github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes/meta/v1"
+	"github.com/pulumi/pulumi-kubernetes/sdk/go/kubernetes/providers"
+	"github.com/pulumi/pulumi/sdk/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+
+		gkeVersion := "1.15.11-gke.1"
```

Comment:
It would be good to pull the latest version here, or this example will break in a few months. Something like this should work:
https://github.com/pulumi/examples/blob/13d379e744efacb971a984b77e5ded6615412ed3/gcp-ts-gke-hello-world/index.ts#L10-L11

Reply:
I just defaulted to latest :)